### PR TITLE
Launching the blog

### LINKED ISSUE #33: Launching the blog
- [ ] Remove dummy data
- [ ] Connect with prose.io
- [ ] Give credentials to media unit 
- [ ] Launch the blog
- [ ] Add nav bar

### PR DESCRIPTION
**Pull request by Hasthikagirihagama**
## Purpose

The purpose of this PR is to fix #33 

## Goals

Complete the following tasks
 - Remove dummy data
 - Connect with prose.io
 - Give credentials to media unit 
 - Launch the blog
 - Add navbar

## Approach

- [x] Remove dummy data
- [x] Connect with prose.io
- [x] Give credentials to media unit 
- [x] Launch the blog
- [x] Add navbar

### Screenshots

https://user-images.githubusercontent.com/70457601/149011901-a08a11ea-3b21-4529-ac5a-5c2f7c3c3a91.mov

  
### Preview Link
https://hasthikagirihagama.github.io/sack-blog/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+            &nbsp;Computer Fraternity, SACK
+          </div>
+        </div>
+      </div>
     </div>
-</footer>
\ No newline at end of file
+  </footer>
+  <!-- //Footer -->
+
```
Comment (jayasanka-sack) [inline _includes/footer.html]:
```suggestion

```

File: _includes/navigation.html [hunk 2]
```diff
@@ -1,34 +1,61 @@
...
+  </nav>
+  <!-- //Navbar -->
+  </script>
+  
+  
+  
+  
+  
+  
+
```
Comment (jayasanka-sack) [inline _includes/navigation.html]:
```suggestion
  </script>

```

Comment (jayasanka-sack) [reply _includes/navigation.html]: Oh, where this script tag comes from? 🤔

File: _includes/footer.html [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
+              </script>
+            &nbsp;Computer Fraternity, SACK
+          </div>
+        </div>
+      </div>
     </div>
-</footer>
\ No newline at end of file
+  </footer>
+  <!-- //Footer -->
```
Comment (jayasanka-sack) [inline _includes/footer.html]: Alway's add a blank line at the end of files

File: _includes/navigation.html [hunk 4]
```diff
@@ -1,34 +1,54 @@
...
+              <div class="gcse-search"></div>
+            </div>
+        </div>
+      </div>
     </div>
-</nav>
\ No newline at end of file
+  </nav>
+  <!-- //Navbar -->
+  </script>
```
Comment (jayasanka-sack) [inline _includes/navigation.html]: there is no opening <script> tag right? Seems like a mistake.

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Good job @Hasthikagirihagama ! Don't forget to add a new line at the end of the files. ;)

**Comment by Hasthikagirihagama** [comment]
I think now all the changes are done @jayasanka-sack

File: _includes/navigation.html [hunk 5]
```diff
@@ -1,34 +1,54 @@
...
+              <div class="gcse-search"></div>
+            </div>
+        </div>
+      </div>
     </div>
-</nav>
\ No newline at end of file
+  </nav>
+  <!-- //Navbar -->
+
```
Comment (jayasanka-sack) [inline _includes/navigation.html]:
```suggestion

```

File: _includes/navigation.html [hunk 6]
```diff
@@ -1,34 +1,53 @@
...
+            <div class="searchSmall" id="SearchBar" style="display: none">
+              <div class="gcse-search"></div>
+            </div>
+        </div>
+      </div>
     </div>
-</nav>
\ No newline at end of file
+  </nav>
+  <!-- //Navbar -->
```
Comment (jayasanka-sack) [inline _includes/navigation.html]:
```suggestion
  <!-- //Navbar -->

```

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Hasthikagirihagama !

**Pull request merged**